src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs

```diff
@@ -38,12 +26,18 @@ public Task DisposeAsync()
         public async Task Subscriptions_EndToEnd()
         {
             TestContext.WriteLine("Subscription management tests...");
+            string repo1Name = "maestro-test1";
+            string repo2Name = "maestro-test2";
+            string channel1Name = $"SubscriptionEndToEnd_TestChannel1_{Environment.MachineName}";
+            string channel2Name = $"SubscriptionEndToEnd_TestChannel2_{Environment.MachineName}";
+
+            _parameters = await TestParameters.GetAsync();
+            SetTestParameters(_parameters);
 
             string repo1Uri = GetRepoUrl(repo1Name);
             string repo2Uri = GetRepoUrl(repo2Name);
-            string repo3Uri = GetRepoUrl(repo3Name);
             string repo1AzDoUri = GetAzDoRepoUrl(repo1Name);
-            string targetBranch = "master";
+            string targetBranch = $"SubscriptionEndToEnd_TargetBranch_{Environment.MachineName}";
```

Comment:
It doesn't need to be unique globally, it's just designed to prevent issues with local & official tests running at the same time. Everything is deleted at the end of the test case so to have a conflict the test case would need to be running twice on the same machine at the same time, which seems pretty unlikely.